[2022-01-11 20:18:11,587] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_03 scheduled__2022-01-06T00:00:00+00:00 [queued]>
[2022-01-11 20:18:11,712] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_03 scheduled__2022-01-06T00:00:00+00:00 [queued]>
[2022-01-11 20:18:11,713] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:18:11,713] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-01-11 20:18:11,713] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-01-11 20:18:11,965] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_03> on 2022-01-06 00:00:00+00:00
[2022-01-11 20:18:11,999] {standard_task_runner.py:52} INFO - Started process 5680 to run task
[2022-01-11 20:18:12,149] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_03', 'scheduled__2022-01-06T00:00:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpose_5if0', '--error-file', '/tmp/tmp4dccu3uu']
[2022-01-11 20:18:12,206] {standard_task_runner.py:77} INFO - Job 46: Subtask task_03
[2022-01-11 20:18:17,228] {local_task_job.py:211} WARNING - State of this instance has been externally set to None. Terminating instance.
[2022-01-11 20:18:17,330] {process_utils.py:120} INFO - Sending Signals.SIGTERM to group 5680. PIDs of all processes in the group: [5680]
[2022-01-11 20:18:17,343] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 5680
[2022-01-11 20:18:17,351] {process_utils.py:70} INFO - Process psutil.Process(pid=5680, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='20:18:11') (5680) terminated with exit code Negsignal.SIGTERM
